contrib/exporters/secadvisor/mod/mangle_logstatus.go

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"time"
+
+	"github.com/spf13/viper"
+
+	"github.com/skydive-project/skydive/common"
+	awsflowlogs "github.com/skydive-project/skydive/contrib/exporters/awsflowlogs/mod"
```

Comment:
Something to think about: now the secadvisor exporters directly depends on the awsflowlogs exporter. This seems more complex and we should examine whether the separation to different exporters still makes sense.